openssl-build/src/lib.rs

```diff
@@ -15,10 +15,6 @@ pub fn define_version_number_cfg() {
         if openssl_version >= 0x01_01_00_00_0 {
             println!("cargo:rustc-cfg=ossl110");
         }
-
```

Comment:
Any reason not to keep this? I thought all our crates were building fine against both versions of OpenSSL.

Reply:
Removing this doesn't stop our code from working with 1.1.1. Removing it removes the way to determine whether the code was being compiled against 1.1.0 or 1.1.1. The only reason the code cared about that was that ed25519 is only available with 1.1.1